portability-api/src/main/java/org/dataportabilityproject/webapp/JsonKeys.java

```diff
@@ -34,4 +34,6 @@ private JsonKeys(){} /** no instantiation */
   public static final String IMPORT_SERVICE_AUTH_EXISTS = "importServiceAuthExists";
   public static final String IMPORT_AUTH_URL = "importAuthUrl";
   public static final String TOKEN = "token";
+  public static final String XSRF_TOKEN = "XSRF-TOKEN";
```

Comment:
Are these our own custom values or standard values? if they are standard, we should link to a rfc or paper where they are explained.

Reply:
These aren't standard but are the default for the angular app. We can use any names for them as long as we hook them up to the client app component. I'll add them explicitly in the angular app and include a comment here that they should match.